### ISSUE TITLE
Error in bicep script

### ISSUE THREAD

**Issue by venkateshkumar** [microsoft/azurechat#253]
Hi Team,

I followed the steps in azure cli and getting the below error. 

Deployment Error Details:
InvalidTemplateDeployment: The template deployment 'all-resources' is not valid according to the validation procedure. The tracking id is 'a15d2882-55c3-4d86-bbe9-1efafd5217e0'. See inner errors for details.
CustomDomainInvalid: Sub domain name 'azure_chat-dev-form-5q3oma7nt4nny' is not valid.
InvalidServiceName: Service name 'azure_chat-devsearch5q3oma7nt4nny' is invalid: Service name must only contain lowercase letters, digits or dashes, cannot start or end with or contain consecutive dashes and is limited to 60 characters.
CustomDomainInvalid: Sub domain name 'azure_chat-devai5q3oma7nt4nny' is not valid

**Comment by jorupp** [comment]
I don't believe an underscore is allowed in the names - try using a dash instead.
